var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -55,6 +55,7 @@ class Paraview(CMakePackage, CudaPackage):
             values=('native', 'fermi', 'kepler', 'maxwell',
                     'pascal', 'volta', 'turing', 'ampere', 'all', 'none'),
             description='CUDA architecture')
+    variant('debug', default=False, description="Enable debug flags")
```

Comment:
Can you expand on the help a bit. Something like: Set all other debug flags besides build_type, such as VTK_DEBUG_LEAKS